src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -534,12 +534,6 @@ private void leaveTheRoom()
         chatRoom = null;
     }
 
-    /**
-     * Method called by {@link #rolesAndPresence} when new member joins
-     * the conference room.
-     *
-     * @param chatRoomMember the new member that has just joined the room.
-     */
     private void onMemberJoined(@NotNull ChatRoomMember chatRoomMember)
```

Comment:
Why removing javadocs from code? This is already enforced in all js code maybe we need to see whether we can enforce it with checkstyle plugin we use. 

https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-contributing#comments

Reply:
I removed it, because it is no longer correct. I also think we should document what the method does, not which class calls it. I didn't write a new description, because I didn't see anything useful I could add. I added a doc now.